msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -80,9 +80,10 @@ items:
        :::image type="content" source="../msteams-platform/assets/images/teams-file-upload-bot/selection-of-resource.png" alt-text="azure resource selection":::
        
          You can also create a new resource group (select **Create new** > enter resource name > select **OK**)
-
+         
     1. If you've created a new resource group, select the required location from **New resource group location** dropdown list.
-
+       For more information see [Create resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups)
```

Comment:
Link to create resource group is not required here, Line 85 already covers this step and there is a link in the UI. Remove.

Reply:
The link is removed as suggested.